[2024-08-25T08:42:18.928+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-25T08:42:18.960+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_catchup_and_backfill_v2.task1 backfill__2024-08-25T00:00:00+00:00 [queued]>
[2024-08-25T08:42:18.973+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_catchup_and_backfill_v2.task1 backfill__2024-08-25T00:00:00+00:00 [queued]>
[2024-08-25T08:42:18.973+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-08-25T08:42:18.997+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): task1> on 2024-08-25 00:00:00+00:00
[2024-08-25T08:42:19.004+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1808) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-25T08:42:19.006+0000] {standard_task_runner.py:72} INFO - Started process 1833 to run task
[2024-08-25T08:42:19.006+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_catchup_and_backfill_v2', 'task1', 'backfill__2024-08-25T00:00:00+00:00', '--job-id', '83', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_catchup_and_backfill.py', '--cfg-path', '/tmp/tmpou8lewrz']
[2024-08-25T08:42:19.008+0000] {standard_task_runner.py:105} INFO - Job 83: Subtask task1
[2024-08-25T08:42:19.077+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_catchup_and_backfill_v2.task1 backfill__2024-08-25T00:00:00+00:00 [running]> on host ae952724060e
[2024-08-25T08:42:19.221+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='shreyas' AIRFLOW_CTX_DAG_ID='dag_with_catchup_and_backfill_v2' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-08-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2024-08-25T00:00:00+00:00'
[2024-08-25T08:42:19.223+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-25T08:42:19.245+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-25T08:42:19.246+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo This is a simple bash command!']
[2024-08-25T08:42:19.259+0000] {subprocess.py:86} INFO - Output:
[2024-08-25T08:42:19.261+0000] {subprocess.py:93} INFO - This is a simple bash command!
[2024-08-25T08:42:19.261+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-25T08:42:19.290+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-25T08:42:19.290+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_catchup_and_backfill_v2, task_id=task1, run_id=backfill__2024-08-25T00:00:00+00:00, execution_date=20240825T000000, start_date=20240825T084218, end_date=20240825T084219
[2024-08-25T08:42:19.312+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-25T08:42:19.324+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-25T08:42:19.332+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
